packages/slice-machine/server/src/api/state.ts

```diff
@@ -139,4 +142,20 @@ export default async function handler() {
     env,
     warnings,
   };
+};
+
+export default async function handler(): Promise<ServerState> {
+  const state = await getBackendState();
+  const { client, cwd, prismicData, ...frontEnv } = state.env;
+  const frontEndEnv: FrontEndEnvironment = {
+    ...frontEnv,
+    prismicData: {
```

Comment:
do we have to send PrismicData if it's just for sending the env ?
IF it's not many changes, could we change it ?

Reply:
Done :)